index.yaml

```diff
@@ -1,68 +1,49 @@
 tools:
   sendgrid:
     reference: ./sendgrid
-    all: true
   file-summarizer:
     reference: ./file-summarizer
-    all: true
   database:
     reference: ./database
   zoom:
     reference: ./zoom
-    all: true
   atlassian-jira:
     reference: ./atlassian/jira
-    all: true
   wordpress:
     reference: ./wordpress
-    all: true
   excel:
     reference: ./excel
-    all: true
   browser:
     reference: ./browser
-    all: true
   bluesky:
     reference: ./bluesky
-    all: true
   slack:
     reference: ./slack
-    all: true
   linkedin-publishing:
     reference: ./linkedin-publishing
-    all: true
   notion:
     reference: ./notion
-    all: true
   google-docs:
     reference: ./google/docs
-    all: true
   google-gmail:
     reference: ./google/gmail
-    all: true
   google-search:
     reference: ./google/search
-    all: true
   google-sheets:
     reference: ./google/sheets
-    all: true
   gitlab:
     reference: ./gitlab
-    all: true
   images:
     reference: ./images
-    all: true
   outlook-mail:
     reference: ./outlook/mail
-    all: true
   outlook-calender:
     reference: ./outlook/calendar
-    all: true
   github:
     reference: ./github
-    all: true
   time:
     reference: ./time
+    nameOverride: time
```

Comment:
This is something I added to workaround a migration issue that has to deal with tool name change. Normally. under the current logic, if a tool is a bundle it has be to defined as the name `x-bundle`. This breaks the previous naming conversion it had with old agent/workflow. So this is just a way to override the name to not follow the same conversion.